crates/wash-cli/src/drain.rs

```diff
@@ -1,11 +1,23 @@
 use std::collections::HashMap;
+use std::path::Path;
 
 use anyhow::Result;
 use serde_json::json;
 use wash_lib::cli::CommandOutput;
 use wash_lib::drain::Drain;
 
-pub fn handle_command(cmd: Drain) -> Result<CommandOutput> {
+pub fn handle_command(cmd: Drain) -> Result<CommandOutput, anyhow::Error> {
+    if matches!(cmd, Drain::All | Drain::Downloads) {
+        let wasmcloud_pid_path =
+            Path::new(&dirs::home_dir().unwrap()).join(".wash/downloads/wasmcloud.pid");
+        let wadm_pid_path = Path::new(&dirs::home_dir().unwrap()).join(".wash/downloads/wadm.pid");
```

Comment:
Just a note -- since you use `home_dir()` here in both places, maybe you can ensure it's present in one place:

```rust
let home_dir = dirs::home_dir().context("failed to determine home directory")?;
```

That said, it feels like we might actually want to use [functions like `host_pid_file()` here](https://github.com/wasmCloud/wasmCloud/blob/e3d7dcb1b9dca81fb0d84f9858ce3fcaa9093eb5/crates/wash-lib/src/config.rs#L26) -- I'd even be open to adding one for the `wadm` PID file.

Reply:
Got it. So i'll make use of `host_pid_file` plus add a new `wadm_pid_file`

Reply:
Sounds great! I'll do the needful :+1: